presto-native-execution/presto_cpp/main/http/HttpClient.cpp

```diff
@@ -123,8 +125,12 @@ class ResponseHandler : public proxygen::HTTPTransactionHandler {
   ResponseHandler(
       const proxygen::HTTPMessage& request,
       velox::memory::MemoryPool* pool,
-      const std::string& body)
-      : request_(request), body_(body), pool_(pool) {
+      const std::string& body,
+      const std::function<void(int)> reportStatsFunc)
```

Comment:
```
const std::function<void(int)>& reportStatsFunc
reportStatsFunc_(reportStatsFunc)
```

Reply:
do you mean `const std::function<void(int)>&&` ? like @tanjialiang suggested? or something else?

Reply:
No you don't want to pass by const rvalue. remove const here would be nice.